### DISCUSSION TITLE
Adding project board to repo unclear

### DISCUSSION THREAD

**Discussion by BenjaminMichaelis** [community/community#10825] [Projects and Issues] [answered]
When looking from this view of a repo it seems to indicate that we can add a project board by going to the profile page. Then after adding a board

![Project Board view](https://user-images.githubusercontent.com/22186029/151844761-0b7397a2-7a18-420e-aa42-f126b45638a4.jpg)
 

![Repo View](https://user-images.githubusercontent.com/22186029/151844666-e77b2218-6eb4-40e3-9304-59db35c885cd.jpg)

This view is of the repo even after refreshing after creating a repo. The wording seems to indicate that a beta board can be associated with a repo, however, it doesn't show up in this list even after adding a board as shown in the first image.

**Comment by BenjaminMichaelis** [comment, marked as answer]
Update, after checking now, I can now add the project, so maybe it just has a long delay to update the cache or something. 

Update:
After checking on a friends account, this is not an issue isolated to my account. Two workarounds are, either just search directly for the board even if it doesn't show up on recent projects, or add an issue to the board and it will show up on recent projects.

**Reply by DennisSmolek** [reply]
How long was your delay? Its frustrating to see the project in the repo project view but not be able to add issues to it...

**Reply by BenjaminMichaelis** [reply]
@DennisSmolek So if you add an issue to the board it will appear in the recent projects, or you can just search for it by its title. I agree, it does not fit with the expected usage.

**Reply by DennisSmolek** [reply]
> @DennisSmolek So if you add an issue to the board it will appear in the recent projects, or you can just search for it by its title. I agree, it does not fit with the expected usage.

My issue was something else, when I forked a public repo I assumed it would inherit the originals settings. For some reason the "issues" feature was disabled. 

This makes sense why it wouldn't show up, but after reading about delays and not having any kind of feedback, it took an hour or two of bashing my head to figure it out..
